test/e2e/node/security_context.go

```diff
@@ -270,6 +270,6 @@ func testPodSELinuxLabeling(f *framework.Framework, hostIPC bool, hostPID bool)
 	framework.ExpectNoError(err, "Error waiting for pod to run %v", pod)
 
 	content, err = tk.ReadFileViaContainer(pod.Name, "test-container", testFilePath)
-	framework.ExpectNoError(err, "Error reading file via container")
+	framework.ExpectError(err)
```

Comment:
Might be worth adding a comment indicating why we expect an error? Or maybe we can add as an argument to the `ExpectError` function?

Reply:
I initially wanted to, but it's explained in a comment 15 lines above this one
```go
	// Confirm that the same pod with a different MCS
	// label cannot access the volume
```
and it's also clear from the following line that we do not expect the file to be readable.

Anyway, patch updated.